packages/edge-login-ui-rn/src/components/screens/newAccount/NewAccountReviewScreen.js

```diff
@@ -41,6 +42,7 @@ const NewAccountReviewScreenComponent = ({ onDone, theme }: Props) => {
 
   return (
     <ThemedScene paddingRem={[0.5, 0, 0.5, 0.5]}>
+      <BackButton marginRem={[0, 0, 1, 0.5]} enabled={false} />
```

Comment:
before you wrote `marginRem={[0, 0, 1, -0.5]}` here is `marginRem={[0, 0, 1, 0.5]}` it's typo?

Reply:
If the icon isn't there then the margin doesn't matter